src/controlplane/client/client.go

```diff
@@ -2,21 +2,23 @@ package client
 
 import (
 	"fmt"
+	"net/http"
 	"net/url"
 
-	log "github.com/sirupsen/logrus"
-
 	"github.com/newrelic/nri-kubernetes/v2/internal/logutil"
 	"github.com/newrelic/nri-kubernetes/v2/src/client"
 	"github.com/newrelic/nri-kubernetes/v2/src/controlplane/client/connector"
 	"github.com/newrelic/nri-kubernetes/v2/src/prometheus"
+	"github.com/sethgrid/pester"
+	log "github.com/sirupsen/logrus"
```

Comment:
imports order, not a big deal but we usually keep:

(
stdlibrary-imports

other-imports

same-repo imports

)

Reply:
again, this is doen by goimports

Reply:
fixed